.gitignore

```diff
@@ -1,11 +1,348 @@
-/bin
-/obj
-/OrchardCore.Commerce.Tests/bin
-/OrchardCore.Commerce.Tests/obj
-*.csproj.user
-/.vs
-/MoneyDataType/bin
-/MoneyDataType/obj
-*.sln
-AddressDataType/bin
-AddressDataType/obj
```

Comment:
All the .po files are gone now.  I removed the .gitattributes file since it wasn't used before.  Visual Studio added that.  I reverted the .gitIgnore file back to the original and just made the changes that were needed.